apps/rebar/test/rebar_completion_SUITE.erl

```diff
@@ -0,0 +1,68 @@
+-module(rebar_completion_SUITE).
+
+-compile([export_all, nowarn_export_all]).
+
+-include_lib("common_test/include/ct.hrl").
+
+suite() ->
+    [].
+
+all() ->
+    [test_competion_gen, check_bash].
+
+groups() ->
+    [].
+
+init_per_suite(Config) ->
+    Shells = [bash],
+    ComplFile = compl_file(Config),
+    ok = filelib:ensure_dir(ComplFile),
+    [{compl_file, ComplFile}, {shells, Shells} | Config].
+
+end_per_suite(_Config) ->
+    ok.
+
+init_per_testcase(_, Config) ->
+    rebar_test_utils:init_rebar_state(Config, "completion_").
+
+end_per_testcase(_, _Config) ->
+    ok.
+
+%% test cases
+
+test_competion_gen(Config) ->
+    Shells = ?config(shells, Config),
+    ComplFile = ?config(compl_file, Config),
+    lists:foreach(fun(Shell) ->
+                    file:delete(ComplFile),
+                    completion_gen(Config, #{shell=>Shell, file=>ComplFile}),
+                    {Shell, true} = {Shell,filelib:is_file(ComplFile)}
```

Comment:
Might make sense to use `?assertMatch/2` or `?assertEqual/2` here while importing `stdlib/include/assert.hrl`.